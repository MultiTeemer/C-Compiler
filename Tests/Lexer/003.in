1
143
13.4
0.3
.3
1e10
1e-10
1e+10
.3e-1