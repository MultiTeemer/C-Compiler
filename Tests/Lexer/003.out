Integer		1		1		1		1
+------------------------------------------------------------------+
Integer		2		1		143		143
+------------------------------------------------------------------+
Float		3		1		13.4		13.400000
+------------------------------------------------------------------+
Float		4		1		0.3		0.300000
+------------------------------------------------------------------+
Float		5		1		.3		0.300000
+------------------------------------------------------------------+
Float		6		1		1e10		10000000000.000000
+------------------------------------------------------------------+
Float		7		1		1e-10		0.000000
+------------------------------------------------------------------+
Float		8		1		1e+10		10000000000.000000
+------------------------------------------------------------------+
Float		9		1		.3e-1		0.030000
+------------------------------------------------------------------+
